web/js/map-OpenLayers.js

```diff
@@ -144,6 +173,25 @@ function fixmystreet_onload() {
             backgroundHeight: 40,
             backgroundXOffset: -10,
             backgroundYOffset: -35
+        },
+        'small': {
+            externalGraphic: fixmystreet.pin_prefix + "pin-${colour}-small.png",
+            graphicWidth: 24,
+            graphicHeight: 32,
+            graphicXOffset: -12,
+            graphicYOffset: -32,
```

Comment:
If you're happy with my solution to this in 1448fdccbb3a3f744c0a030ea3a13afad565f617 I'll squash it into the previous commit and merge this PR.